src/utils/queryBuilder.js

```diff
@@ -1,10 +1,25 @@
 import _ from 'lodash';
 import moment from 'moment';
 
+
+function tokenizeSearchTerms(queryString) {
+  // split prototype method is not robust enough: https://blog.tompawlak.org/split-string-into-tokens-javascript
+  const tokenizedTerms = queryString.match(/\S+/g);
+
+  if (!tokenizedTerms.length) {
+    return '';
+  }
+
+  const queryStringArray = tokenizedTerms
+    .map(term => `(contains(title, '${term}') or contains(name, '${term}') or contains(description, '${term}'))`);  // eslint-disable-line max-len
```

Comment:
I had the string itself implicitly returning on a separate line and it was still barking :\  I only disabled `max-len` on the line so other rules will still be evaluated